interfaces/builtin/opengl.go

```diff
@@ -56,6 +56,12 @@ const openglConnectedPlugAppArmor = `
   # For now, allow 'c'haracter devices and 'b'lock devices based on
   # https://www.kernel.org/doc/Documentation/devices.txt
   /run/udev/data/c226:[0-9]* r,  # 226 drm
+
+  # This is a fix for access denied when opengl interface is used with
+  # framebuffer interface in the same snap.
+  # Fix for https://bugs.launchpad.net/snapd/+bug/1675738
+  /dev/fb[0-9]* rw,
```

Comment:
The opengl doesn't allow access to /dev/fb* at the moment. See https://github.com/snapcore/snapd/blob/master/interfaces/builtin/opengl.go

If you're using the framebuffer interface together with the opengl interface at the moment then you have /dev/fb in the resulting profile as it comes in from the framebuffer interface. In this case also udev tagging is active which prevents you from accessing any other node in /dev

The goal of this PR is to allow you using just the opengl interface and drop the framebuffer interface. This will get you away from having udev tagging in the way. In a long term fix we will convert all interfaces to use proper udev tagging.